src/drivers/intel/dmic/dmic_computed.c

```diff
@@ -67,7 +67,7 @@ static void find_modes(struct dai *dai,
 
 	/* Check for sane pdm clock, min 100 kHz, max ioclk/2 */
 	if (dmic->global->prm[di].pdmclk_max < DMIC_HW_PDM_CLK_MIN ||
-	    dmic->global->prm[di].pdmclk_max > DMIC_HW_IOCLK / 2) {
+	    dmic->global->prm[di].pdmclk_max > CONFIG_DMIC_HW_IOCLK / 2) {
```

Comment:
Is there a PR to do this in Zephyr DMIC too ?

Reply:
Zephyr DMIC already has configurable HW_IO_CLOCK (CONFIG_DAI_DMIC_HW_IOCLK).